src/main/scala/com/databricks/sparkdl/ImageUtils.scala

```diff
@@ -113,13 +115,15 @@ private[sparkdl] object ImageUtils {
    * @param tgtChannels number of channels of output image (must be 3), may be used later to
    *                    support more channels.
    * @param spImage     image to resize.
+   * @param scaleHint   hint which algorhitm to use, see java.awt.Image#SCALE_DEFAULT
```

Comment:
This suggests that our default is `SCALE_DEFAULT`, but I don't think we do.

Reply:
btw, I think `SCALE_DEFAULT` might be implementation specific for java.awt.Image subclasses.

Reply:
hmm I randomly picked one, it is not supposed to suggest what the default value is. I can replace it with default.